app/1.0/toolbox/news-case-study.md

```diff
@@ -0,0 +1,264 @@
+user experience.
+
+## App Structure
+
+Like other PWAs, News is designed with an [application shell architecture](https://developers.google.com/web/fundamentals/architecture/app-shell). The app shell element, `<news-app>`, contains the main framework for News. Here's the basic structure:
+
+<div class="image-container layout horizontal">
+  <div class="image-wrapper">
+    <img src="app structure.jpg" alt="app structure" width="600px">
+  </div>
+</div>
+
+* `<news-app>` lays out the structure for the application.
+* `<app-route>` contains routing logic that parses the user's selected URL.
+* `<news-data>` fetches resources from the raw HTML files and images in the /data/ folder and its subfolders.
+* `<iron-pages>` switches between the News application's two views - list view and article view.
+* `<news-list>` is the application's *list view*.
+* `<news-article>` is the application's *article view*.
+
+<div class="image-container layout horizontal">
+  <div class="image-wrapper">
+    <img src="news-list.jpg" alt="screenshot of the list view" width="300px">
+  </div>
+  <div class="image-wrapper">
+    <img src="news-article.jpg" alt="screenshot of the article view" width="300px">
+  </div>
+</div>
+
+## Views and routing
+
+Views in the News app use the same implementation as the Shop app. [See the Shop app case study for more information on views and routing](https://www.polymer-project.org/1.0/toolbox/case-study#views).
+
+For more information on encapsulated routing, see [Encapsulated routing with elements](https://www.polymer-project.org/1.0/blog/routing).
+
+For more information on the `<app-route>` Polymer element used in these implementations, see the [`<app-route>` API reference](https://elements.polymer-project.org/elements/app-route).
+
+There are two main views in the News app. The `<news-list>` element displays a list of articles for the selected category - for example, Top Stories. The `<news-article>` element displays article content. When the user is offline and the category data is not cached, both views display a network error via the `<news-network-warning>` element:
+
+`news-list.html`
+```html
+<news-network-warning
+    hidden$="[[!failure]]"
+    offline="[[offline]]"
+    on-try-reconnect="_tryReconnect"></news-network-warning>
+```
+
+If the user is offline but category data is cached and ServiceWorker is installed,
+the cached content is displayed.
+
+-   [`<app-route>` API reference](https://elements.polymer-project.org/elements/app-route)
+
+## Routing and Data Binding in the News App
+
+Routing and data bindings work together in the News app to retrieve and display the data relevant to the URL that the user selects.
+
+Polymer's data system allows for data to flow one-way (downward-only, from host element to target element) or two-way (from host to target and target to host). For more information on data binding in Polymer, see [fkjsdkfjl].
```

Comment:
While fkjsdkfjl is a reputable source on the subject of data binding, I think I'd go for the more canonical `[Data system concepts](../docs/devguide/data-system)` or maybe `[Data binding](../docs/devguide/data-binding)`.